### PR TITLE
WBCAMS-729 search common job titles

### PR DESCRIPTION
**Pull request by rjkuyvenhoven**

### CODE REVIEW

File: src/config/sync/core.entity_view_display.node.career_profile.career_profile_search_index.yml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+targetEntityType: node
+bundle: career_profile
+mode: career_profile_search_index
+content:
+  extra_field_career_videos_introduction:
+    settings: {  }
+    third_party_settings: {  }
+    weight: 9
+    region: content
+  extra_field_employment_introduction:
```
Comment (infojunkie) [inline src/config/sync/core.entity_view_display.node.career_profile.career_profile_search_index.yml:41]: we don't need this field

File: src/config/sync/core.entity_view_display.node.career_profile.career_profile_search_index.yml [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    - field.field.node.career_profile.field_skills_2
+    - field.field.node.career_profile.field_tags
+    - field.field.node.career_profile.field_work_environment
+    - node.type.career_profile
+id: node.career_profile.career_profile_search_index
+targetEntityType: node
+bundle: career_profile
+mode: career_profile_search_index
+content:
+  extra_field_career_videos_introduction:
```
Comment (infojunkie) [inline src/config/sync/core.entity_view_display.node.career_profile.career_profile_search_index.yml:36]: we don't need this field

**Review by infojunkie — CHANGES_REQUESTED** [review]
Some unneeded fields slipped in there

**Comment by rjkuyvenhoven** [comment]
> Some unneeded fields slipped in there

I'm not sure why those fields are showing there. When I check the Manage Display "Search Index Job Titles" is the only field configured to be displayed.

During testing I set the view to display the rendered content and only the Job Titles were displayed.

**Comment by infojunkie** [comment]
I've manually moved those fields - can you please check nothing is messed up?

**Comment by rjkuyvenhoven** [comment]
I have tested and search seems to be working as expected.

**Review by infojunkie — APPROVED** [review]
Good to go, thanks!

**Pull request merged**